### PR TITLE
Convert the landing page to our new design

### PR DESCRIPTION
**Pull request by jankeromnes**
This PR is dependent on https://github.com/JanitorTechnology/janitor/pull/273, so wait until that one merges first.

Let's convert Janitor's front page to our new design, and make it look cleaner / friendlier / more fun / more exciting, as requested in https://discourse.janitor.technology/t/new-janitor-front-page/119

TODO:
- [x] Random `<verb> <action> faster` tagline
- [x] Give an exciting glimpse of "what problems does it solve for me?" (i.e. highlight the most valuable benefits of using Janitor)
- [ ] Give an exciting glimpse of "what does it look like?"
- [x] Buttons that call to action (e.g. `Get started` or `Free account` or better options)
- [ ] [Personality](https://avc.com/2011/09/minimum-viable-personality/)
- [x] Sign up form
- [x] Section with partner logos (IRILL, Mozilla, Cloud9, Datadog)
- [x] Section with user feedback from https://discourse.janitor.technology/t/happy-janitor-quotes/151 (random quote or carousel-like)

### CODE REVIEW

**Comment by jankeromnes** [comment]
Wow, our very fist Self API handler timeout. Let's see if it reproduces after a Travis job restart.

**Comment by notriddle** [comment]
This gets very, very broken when I resize the browser window. The quotes list should wrap, and the 100vh high containers get too narrow when I shrink the height of the browser.

**Comment by notriddle** [comment]
As for the actual ad copy, can I just ask it to *just say what the thing is*? The video and tagline should go first, sure, but below the quotes, just say this:

---

That's what Janitor, an on-demand development environment service for Free and Open Source projects, can do for you. You click a button next to a piece of software that you want to work on, and we start up a virtual Linux computer (a "container") that's already set up for doing development on it. Every container:

* Ships with all the project's dependencies pre-installed.
* Has the source code you need already downloaded and compiled.
* Is fast, and connected to a powerful backbone internet connection.
* Can be accessed anywhere you have an internet connection and a web browser; even a library computer or a Chromebook.
* Runs Linux (Windows support is coming soon).
* Comes with convenience scripts and utilities that the maintainers themselves use to improve their productivity.
* Gets completely isolated from your own computer, and from your other containers.
* Starts in a few seconds.

**Comment by arshdkhn1** [comment]
For the quotes section, I think we should also add the position/portfolio of people who have given quotes. It might be more effective.

**Comment by arshdkhn1** [comment]
time to record new intro video. @jankeromnes if you are ok with this suggestion then let me know before you record new video.

**Comment by arshdkhn1** [comment]
I have styled quote section. 
image
One problem that I was facing  was that the iframe height is not 56.25%(padding-top on parent wrapper) on large screen. I tried to find out why is that happeing but couldn't do it in time. So if anyone solves that then do let me know.

File: lib/routes.js [hunk 1]
```diff
@@ -119,6 +119,65 @@ exports.landingPage = function (response, user = null) {
...
+  './templates/projects-new.html',
+]);
+const landingTaglines = [
+  'Fix bugs',
+  'Review code',
+  'Migrate code',
+  'Refactor code',
+  'Fix technical debt',
+  'Investigate bugs',
+  'Debug stuff',
```
Comment (PTrottier) [inline lib/routes.js]: I am not a huge advocate for the word stuff, in this case, I think "Debug with ease" and "Implement that dream" would sound better.

File: templates/landing-new.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+              <li><strike>Long build times</strike></li>
+              <li><strike>Requires high-end hardware</strike></li>
+              <li>Cloud-based</li>
+            </ul>
+          </div>
+          <div class="grow">
+            <h1>Some of our awesome features</h1>
+            <ul>
+              <li>Online IDE with review comments sidebar</li>
+              <li>???</li>
```
Comment (PTrottier) [inline templates/landing-new.html]: Idea: 

- Expertly crafted environments ready in seconds and on demand

File: lib/routes.js (hunk 1 above)
Comment (jankeromnes) [reply lib/routes.js]: Great suggestion! I'm not too fond of the word "stuff" either... But note that the word " faster" gets appended after every tagline (and "Debug with ease faster" sounds a bit strange).

**Comment by jankeromnes** [comment]
Thank you so much everyone for your contributions! I feel like this page is shaping up extremely nice, much better as a collaborative effort than I could have achieved just on my own.

I'll try to put in some more work this week, and bring it closer to completion. I don't want to stand in the way of your fantastic new design forever! 😋

File: lib/routes.js (hunk 1 above)
Comment (jankeromnes) [reply lib/routes.js]: How about "Debug things faster"? Or just "Debug faster"? Also I'll add "Implement that dream faster", sounds great!

File: templates/landing-new.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            <blockquote>This will change the lives of a lot of contributors.</blockquote>
+            <span>― Michael Kohler</span>
+          </div>
+          <div class="quote grow">
+            <blockquote>Mind blown! This would have literally saved months of my life years ago when I hacked on Firefox.</blockquote>
+            <span>― Brian King</span>
+          </div>
+        </div>
+        <div class="container">
+          <h2>In one click, spin up complete development environments:</h2>
```
Comment (ishitatsuyuki) [inline templates/landing-new.html]: Missing `that` here?

File: templates/landing-new.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+          <div class="grid">{{
+            for (let id in projects) {
+              const project = projects[id];
+            }}
+            <div class="card">
+              <img class="card-image" src="{{= project.icon in xmlattr}}" alt="{{= project.name in xmlattr}} Logo">
+              <div class="card-content">
+                <div class="grow">
+                  <h2>{{= project.name in html}}</h2>
+                </div>
```
Comment (ishitatsuyuki) [inline templates/landing-new.html]: We should display the project description as well.

**Comment by ishitatsuyuki** [comment]
Lighthouse recommended to use >=16px for text, in order to achieve best SEO. Please consider changing to it.

File: templates/landing-new.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        <div class="container">
+          <h2>Get started now:</h2>
+          <div class="form-control-container">
+            <input class="form-control large" name="email" placeholder="you@yourdomain.com" spellcheck="false" autocapitalize="off" type="email" autocomplete="off" required>
+            <button class="btn btn-primary large">Log In</button>
+          </div>
+        </div>
+        <div class="container">
+          <h2>Our awesome partners</h2>
+          <div class="partners">
```
Comment (ishitatsuyuki) [inline templates/landing-new.html]: I believe this is overflowing viewport.

**Comment by ishitatsuyuki** [comment]
@nitm, mind checking out the things I commented?

File: templates/landing-new.html (hunk 3 above)
Comment (nt1m) [reply templates/landing-new.html]: "spin up complete development environments" is correct english :)

File: templates/landing-new.html (hunk 4 above)
Comment (nt1m) [reply templates/landing-new.html]: I'm not too sure about this. The goal of showing projects on the landing page is just to impress people with the projects they can contribute to, so the description is not necessarily useful towards that goal.

File: templates/landing-new.html (hunk 5 above)
Comment (nt1m) [reply templates/landing-new.html]: Thanks! I'll fix that

File: templates/landing-new.html (hunk 3 above)
Comment (ishitatsuyuki) [reply templates/landing-new.html]: No, I mean that it doesn't smoothly get concatenated with the things after that. It sounds like "In one click, spin up complete development environments: Run Linux", that's why I said it sounds like missing "that".

File: templates/landing-new.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+            <a href="https://mozilla.org" class="partner-logo"><img src="/img/partners/mozilla.svg"></a>
```
Comment (ishitatsuyuki) [inline templates/landing-new.html]: Please add alts.

File: templates/landing-new.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
+        <div class="container">
+          <div class="text-center">
+            {{
+              const tagline = taglines[Math.floor(Math.random() * taglines.length)] + ' faster';
+            }}<h1 class="heading">{{= tagline in html}}</h1>
+            <h2 class="grey-text description">The fastest development system in the world.</h2>
+          </div>
+          <div id="intro-video">
+            <iframe src="https://www.youtube.com/embed/5sNDMIh-iVw" allowfullscreen></iframe>
```
Comment (ishitatsuyuki) [inline templates/landing-new.html]: Please add `title` to the iframe.

**Comment by notriddle** [comment]
Rebased onto master and added the alt text.

**Comment by jankeromnes** [comment]
Rebased onto master and made a few tweaks

**Comment by jankeromnes** [comment]
@notriddle what do you think of https://github.com/JanitorTechnology/janitor/pull/266/commits/beb0f497d5612f8a9309fc1f70acf89c46fc2b83 ?

**Comment by notriddle** [comment]
Looks pretty good. In particular, I agree that we should be consistent about the colon usage, but I'd prefer standardize on never using colons.

**Comment by jankeromnes** [comment]
I agree that no `:` at the end of titles looks much cleaner. 😄 Thanks!

**Comment by jankeromnes** [comment]
@ishitatsuyuki Could you please verify that all your remarks have been addressed, and approve this pull request if they are?

**Comment by jankeromnes** [comment]
Woohoo! Thanks all! :fireworks: :rocket: :boom:

**Pull request merged**